TCD-2585 (#1590)
saucelabs/sauce-docs: docs/api-testing/composer/assertion-components.md

| Stop test if fails | Yes, No | No |

* __Expression__: The path to the element we want to operate on (e.g., `payload.ProductID`). See [Expression](/api-testing/composer/expressions/) for more details.
* __Value__: The value we want to compare the expression to.
* __Type__: The type of the value. **Auto** means the engine will try to identify the type of the value.
* __Comment__: See [Comment](#comment).
* __Level__: See [Level](#level).
* __Modifier__: The assertion is considered verified if it does not pass.
* __Execute if item exists__: The assertion is evaluated only if the element exists. This is useful when the element does not always exist.

> * __Stop test if fails__: The test will be immediately stopped if the assertion fails.

Reviewer: ```suggestion
* __Stop test if fails__: The test will be immediately stopped if the assertion fails. See [Stop if Test Fails](#stop-if-test-fails).
```

| Stop test if fails | Yes, No | No |

> * __Expression__: The path to the element we want to operate on (e.g., `payload.ProductID`). See [Expression](/api-testing/composer/expressions/) for more details.

Reviewer: ```suggestion
* __Expression__: The path to the element we want to operate on (e.g., `payload.ProductID`). See [Using Expressions in Test Components](/api-testing/composer/expressions/) for more details.
```

* __Value__: The value we want to compare the expression to.
* __Type__: The type of the value. **Auto** means the engine will try to identify the type of the value.
* __Comment__: See [Comment](#comment).
* __Level__: See [Level](#level).
* __Modifier__: The assertion is considered verified if it does not pass.
* __Execute if item exists__: The assertion is evaluated only if the element exists. This is useful when the element does not always exist.

> * __Stop test if fails__: The test will be immediately stopped if the assertion fails.

Reviewer: ```suggestion
* __Stop test if fails__: The test will be immediately stopped if the assertion fails. See [Stop if Test Fails](#stop-if-test-fails).
```

| Stop test if fails | Yes, No | No |

> * __Expression__: The path to the element we want to operate on (e.g., `payload.ProductID`). See [Expression](/api-testing/composer/expressions/) for more details.

Reviewer: ```suggestion
* __Expression__: The path to the element we want to operate on (e.g., `payload.ProductID`). See [Using Expressions in Test Components](/api-testing/composer/expressions/) for more details.
```

> * __Expression__: The path to the element we want to operate on (e.g., `payload.ProductID`). See [Expression](/api-testing/on-prem/reference/expression/) for more details.

Reviewer: ```suggestion
* __Expression__: The path to the element we want to operate on (e.g., `payload.ProductID`). See [Using Expressions in Test Components](/api-testing/composer/expressions/) for more details.
```

* __Value__: The value we want to compare the expression to.
* __Comment__: See [Comment](#comment).
* __Level__: See [Level](#level).
* __Modifier__: The assertion is considered verified if it does not pass.
* __Execute if item exists__: The assertion is evaluated only if the element exists. This is useful when the element does not always exist.

> * __Stop test if fails__: The test will be immediately stopped if the assertion fails.

Reviewer: ```suggestion
* __Stop test if fails__: The test will be immediately stopped if the assertion fails. See [Stop if Test Fails](#stop-if-test-fails).
```

| Stop test if fails | Yes, No | No |

> * __Expression__: The path to the element we want to operate on (e.g., `payload.ProductID`). See [Expression](/api-testing/composer/expressions/) for more details.

Reviewer: ```suggestion
* __Expression__: The path to the element we want to operate on (e.g., `payload.ProductID`). See [Using Expressions in Test Components](/api-testing/composer/expressions/) for more details.
```

* __Value__: The value we want to compare the expression to.
* __Comment__: See [Comment](#comment).
* __Level__: See [Level](#level).
* __Modifier__: The assertion is considered verified if it does not pass.
* __Execute if item exists__: The assertion is evaluated only if the element exists. This is useful when the element does not always exist.

> * __Stop test if fails__: The test will be immediately stopped if the assertion fails.

Reviewer: ```suggestion
* __Stop test if fails__: The test will be immediately stopped if the assertion fails. See [Stop if Test Fails](#stop-if-test-fails).
```

| Stop test if fails | Yes, No | No |

> * __Expression__: The path to the element we want to operate on (e.g., `payload.ProductID`). See [Expression](/api-testing/composer/expressions/) for more details.

Reviewer: ```suggestion
* __Expression__: The path to the element we want to operate on (e.g., `payload.ProductID`). See [Using Expressions in Test Components](/api-testing/composer/expressions/) for more details.
```

    * _email_: checks if field is a valid email format;
    * _map_: checks if field is a map type;
    * _array_: checks if the field is an array.
* __Comment__: See [Comment](#comment).
* __Level__: See [Level](#level).
* __Modifier__: The assertion is considered verified if it does not pass.
* __Execute if item exists__: The assertion is evaluated only if the element exists. This is useful when the element does not always exist.

> * __Stop test if fails__: The test will be immediately stopped if the assertion fails.

Reviewer: ```suggestion
* __Stop test if fails__: The test will be immediately stopped if the assertion fails. See [Stop if Test Fails](#stop-if-test-fails).
```

> * __Expression__: The path to the element we want to operate on (e.g., `payload.ProductID`). See [Expression](/api-testing/composer/expressions/) for more details.

Reviewer: ```suggestion
* __Expression__: The path to the element we want to operate on (e.g., `payload.ProductID`). See [Using Expressions in Test Components](/api-testing/composer/expressions/) for more details.
```

* __Value__: The value we want to compare the expression to.
* __Comment__: See [Comment](#comment).
* __Level__: See [Level](#level).
* __Modifier__: The assertion is considered verified if it does not pass.
* __Execute if item exists__: The assertion is evaluated only if the element exists. This is useful when the element does not always exist.

> * __Stop test if fails__: The test will be immediately stopped if the assertion fails.

Reviewer: ```suggestion
* __Stop test if fails__: The test will be immediately stopped if the assertion fails. See [Stop if Test Fails](#stop-if-test-fails).
```

| Stop test if fails | Yes, No | No |

> * __Expression__: The path to the element we want to operate on (e.g., `payload.ProductID`). See [Expression](/api-testing/composer/expressions/) for more details.

Reviewer: ```suggestion
* __Expression__: The path to the element we want to operate on (e.g., `payload.ProductID`). See [Using Expressions in Test Components](/api-testing/composer/expressions/) for more details.
```

    * _country codes_: checks if the field contains a valid country code (i.e., 'US', 'FR', 'DK');
    * _currency codes_: checks if the fields is a valid currency (i.e., 'USD', 'EUR');
* __Regex value__: Specify the regular expression you want to use for checking the expression.
* __Comment__: See [Comment](#comment).
* __Level__: See [Level](#level).
* __Modifier__: The assertion is considered verified if it does not pass.
* __Execute if item exists__: The assertion is evaluated only if the element exists. This is useful when the element does not always exist.

> * __Stop test if fails__: The test will be immediately stopped if the assertion fails.

Reviewer: ```suggestion
* __Stop test if fails__: The test will be immediately stopped if the assertion fails. See [Stop if Test Fails](#stop-if-test-fails).
```

| JsonSchema | JSON schema definition | Yes |
| Comment | String | No |

> * __Expression__: The path to the element we want to operate on (e.g., `payload.ProductID`). See [Expression](/api-testing/composer/expressions/) for more details.

Reviewer: ```suggestion
* __Expression__: The path to the element we want to operate on (e.g., `payload.ProductID`). See [Using Expressions in Test Components](/api-testing/composer/expressions/) for more details.
```
